src/components/Match/matchPages.jsx

```diff
@@ -232,9 +232,12 @@ const matchPages = [{
 }, {
   name: strings.tab_cosmetics,
   parsed: true,
-  content: match => (<div>
-    <TeamTable match={match} columns={cosmeticsColumns} heading={strings.heading_cosmetics} />
-  </div>),
+  content: match => (
+    <div>
+      <Heading title={strings.heading_cosmetics} />
```

Comment:
Actually cosmetics IMO shouldn't be splitted for teams as other tables. It's not a team info, its personal.